### PR TITLE
DOCSP-20400 replica sets can connect using Socks5

### PR DESCRIPTION
**Pull request by jocelyn-mendez1**
Replica sets can connect using Socks5

JIRA: https://jira.mongodb.org/browse/DOCSP-20400?filter=-1

STAGING (Release notes): https://docs-mongodbcom-staging.corp.mongodb.com/compass/docsworker-xlarge/DOCSP-20400/release-notes/

(Connect Page): https://docs-mongodbcom-staging.corp.mongodb.com/compass/docsworker-xlarge/DOCSP-20400/connect/#connect-2

### CODE REVIEW

**Comment by jocelyn-mendez1** [comment]
Hi @jeff-allen-mongo, I added the note regarding the replica sets and Socks5. Let me know of any additional changes that can be made. Thanks!

File: source/release-notes.txt [hunk 1]
```diff
@@ -42,7 +42,7 @@ Bug Fixes:
 
 - BSON Transpilers: Account for bson Decimal128 validation changes.
 
-- Make SSH tunnel use Socks5.
+- Make SSH tunnel use Socks5. You can connect to replica sets using Socks5.
```
Comment (jeff-allen-mongo) [inline source/release-notes.txt:45]: I'm not sure that users will be using Socks5. Instead, I think this is something that Compass enables under the hood that enables users to connect to replica sets using SSH. Therefore, we might want to change this sentence to something like:

"You can now connect to replica sets (and sharded clusters? not sure if this is true. Need confirmation from tech) using an SSH tunnel."

**Review by jeff-allen-mongo — CHANGES_REQUESTED** [review]

**Comment by jeff-allen-mongo** [comment]
Thanks @jocelyn-mendez1 ! Left a comment for you to consider.

File: source/release-notes.txt (hunk 1 above)
Comment (jocelyn-mendez1) [reply source/release-notes.txt]: Done and confirmed shared clusters with tech.

**Comment by jocelyn-mendez1** [comment]
Hi @jeff-allen-mongo, I was able to confirm that shared clusters can also be connected to using SSH tunnels. Let me know of any other possible changes. Thanks!

File: source/release-notes.txt [hunk 2]
```diff
@@ -42,7 +42,8 @@ Bug Fixes:
 
 - BSON Transpilers: Account for bson Decimal128 validation changes.
 
-- Make SSH tunnel use Socks5.
+- Make SSH tunnel use Socks5. You can now connect to replica sets and 
+  shared clusters using an SSH tunnel.
```
Comment (jeff-allen-mongo) [inline source/release-notes.txt:46]:
```suggestion
  sharded clusters using an SSH tunnel.
```

**Review by jeff-allen-mongo — APPROVED** [review]
LGTM mod typo fix. Thanks @jocelyn-mendez1!

File: source/release-notes.txt (hunk 2 above)
Comment (jocelyn-mendez1) [reply source/release-notes.txt]: Done.

**Review by addaleax — APPROVED** [review]

**Pull request merged**